features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64Serializer.kt

```diff
@@ -139,45 +167,17 @@ internal class Base64Serializer private constructor(
         isBitmapPoolRegisteredForCallbacks = true
     }
 
+    private fun compressBitmap(bitmap: Bitmap): ByteArray {
```

Comment:
very minor: seems like there is no need for this method, all call sites can use `webPImageCompression.compressBitmap` directly.